Airflow 3.0.0 Announcement (#1166)
apache/airflow-site: landing-pages/site/content/en/blog/airflow-three-point-oh-is-here/index.md

---
title: "Apache Airflow® 3 is Generally Available!"
linkTitle: "Apache Airflow 3.0 is here!"
authors:
  - name: "Kaxil Naik"
    github: "kaxil"
    linkedin: "kaxilnaik"
  - name: "Vikram Koka"
    github: "vikramkoka"
    linkedin: "vikramkoka"
description: "We're proud to announce that Apache Airflow 3.0.0 has been released."
tags: [Release]
date: "2025-04-22"
---

> We announced our intent to focus on Apache Airflow 3.0® as the next big milestone for the Airflow project at the Airflow Summit in September 2024. I am delighted to announce that Airflow 3.0 is now released!

Reviewer: ```suggestion
We announced our intent to focus on Apache Airflow 3.0® as the next big milestone for the Airflow project at the Airflow Summit in September 2024. We are delighted to announce that Airflow 3.0 is now released!
```

Should it be "we"?

> We announced our intent to focus on Apache Airflow 3.0® as the next big milestone for the Airflow project at the Airflow Summit in September 2024. I am delighted to announce that Airflow 3.0 is now released!

Reviewer: ```suggestion
We announced our intent to focus on Apache Airflow 3.0® as the next big milestone for the Airflow project at the Airflow Summit in September 2024. We are delighted to announce that Airflow 3.0 is now released!
```

Should it be "we"?

Author: Yup, updated
